packages/jest-config/src/normalize.ts

```diff
@@ -1220,6 +1220,19 @@ export default async function normalize(
     newOptions.logHeapUsage = false;
   }
 
+  if (argv.shard) {
+    const [shardIndex, shardCount] = argv?.shard
+      .split('/')
+      .filter(d => /^\d+$/.test(d))
+      .map(d => parseInt(d, 10))
+      .filter((shard: number) => !Number.isNaN(shard));
+
+    newOptions.shard = {
```

Comment:
should we verify `shardIndex <= shardCount`? and that both are greater than `0`?

Reply:
Yes I think that makes sense - it duplicates the validation logic in `packages/jest-cli/src/cli/args.ts`, is that acceptable?

Reply:
I think validating only here in `normalize` makes sense - the rest of Jest (at least "internal" packages like `jest-cli`) should trust the data structure of a normalized config object.